src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1004,6 +1004,19 @@ namespace winrt::TerminalApp::implementation
                 items.push_back(profileItem);
                 break;
             }
+            case NewTabMenuEntryType::Action:
+            {
+                const auto actionEntry = entry.as<ActionEntry>();
+                const auto actionId = actionEntry.ActionId();
+                if (_settings.ActionMap().GetActionById(actionId) == nullptr)
+                {
+                    break;
+                }
+
+                auto actionItem = _CreateNewTabFlyoutAction(actionId);
+                items.push_back(actionItem);
```

Comment:
```suggestion
                if (_settings.ActionMap().GetActionById(actionId) )
                {
                    break;
                    auto actionItem = _CreateNewTabFlyoutAction(actionId);
                    items.push_back(actionItem);
                }
```
nit: you can consolidate this a bit